### PR TITLE
chore(init-templates): Python app template imports 'core' as 'cdk' to align sample code across languages

### PR DESCRIPTION
**Pull request by ghost**
For consistency and to make it easier to translate TypeScript examples into Python, import the `aws_cdk.core` module as `cdk`.

There are snippets in the CDK Developer Guide that assume the `core` module is imported as `core`. It is difficult to synchronize changes to the CDK with changes in the Guide, so this PR causes the `core` module to be imported under _both_ names. Once this change is released, the Guide snippets can be updated to use `cdk` rather than `core` to refer to the `core` module. At some point in the future, this template can be updated again to remove the now-redundant `core` import.

This is to support this Guide issue: https://github.com/awsdocs/aws-cdk-guide/issues/268 which intends to make all code snippets in the Guide follow the guidance in the "Working with the CDK in Python" topic.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/aws-cdk/lib/init-templates/v1/app/python/app.template.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env python3
 
-from aws_cdk import core
+from aws_cdk import core, core as cdk
```
Comment (shivlaks) [inline packages/aws-cdk/lib/init-templates/v1/app/python/app.template.py:3]:
```suggestion
import core from aws_cdk as cdk
```

**Review by shivlaks — CHANGES_REQUESTED** [review]

File: packages/aws-cdk/lib/init-templates/v1/app/python/%name.PythonModule%/%name.PythonModule%_stack.template.py [hunk 2]
```diff
@@ -1,9 +1,8 @@
-from aws_cdk import core
+from aws_cdk import core, core as cdk
```
[the remark this answers is not in the document]
Comment (Jerry-AWS) [reply packages/aws-cdk/lib/init-templates/v1/app/python/%name.PythonModule%/%name.PythonModule%_stack.template.py:1]: This isn't valid Python syntax and doesn't have the intended effect (having `core` imported a both `cdk` and `core` to give me a window to update the code snippets to use `cdk`, where the examples can use either without breaking).

File: packages/aws-cdk/lib/init-templates/v1/app/python/app.template.py (hunk 1 above)
Comment (Jerry-AWS) [reply packages/aws-cdk/lib/init-templates/v1/app/python/app.template.py:3]: Same as above.

**Comment by Jerry-AWS** [comment]
I changed this to do the two imports of `core` separately and added a comment to explain why it was being imported twice.

**Review by rix0rrr — APPROVED** [review]

**Comment by Jerry-AWS** [comment]
@rix0rrr How do I fix the PR Linter error? I don't think there are any tests that involve the template. I can report that the template does synth fine.

**Pull request merged**